sklearn/feature_selection/from_model.py

```diff
@@ -169,6 +169,12 @@ def fit(self, X, y=None, **fit_params):
         self.estimator_.fit(X, y, **fit_params)
         return self
 
+    @property
+    def threshold_(self):
+        scores = _get_feature_importances(self.estimator_, self.norm_order)
```

Comment:
This can be expensive. So where a user used to be able to use `sfm.threshold_` with impunity, now it may take some time.